LiskHQ/lisk-desktop
# Wallet - Export Transactions - closes #2666

### What issue have I solved?

-- #2666 

### How have I implemented/fixed it?

- [x] Implement a button next to the filters in the transaction table in wallet page.
- [x] csv link for download the transactions is pending from service

**NOTE:** Right now ONLY have the link for testnet stagin mode there is not no mainnet so far and waiting for final version of testnet.

### How has this been tested?


1. Login to the app.
2. Go to wallet see that there are transactions.
3. Do click in the Export to CSV buttons in the transactions table.
4. A csv file should be downloaded in your computer and after open it should be properly formated

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
